src/main/java/org/fcrepo/spec/testsuite/authz/WebACModes.java

```diff
@@ -104,9 +147,18 @@ public void writeAllowedPUT(final String uri) {
     @Parameters({"param1"})
     public void writeAllowedPOST(final String uri) {
         final TestInfo info = setupTest("5.0-J", "writeAllowedPOST",
-                "acl:Write gives access to a class of operations that can modify the resource. In a REST API " +
-                        "context, this would include POST.",
-                "https://fedora.info/2018/06/25/spec/#resource-authorization", ps);
+                                        "acl:Write gives access to a class of operations that can modify the resource" +
+                                        ". In a REST API " +
+                                        "context, this would include POST.",
+                                        "https://fedora.info/2018/06/25/spec/#resource-authorization", ps);
+
+        //create a resource
+        final String resourceUri = createResource(uri, info.getId());
+        final String aclUri = createAclForResource(resourceUri, "user-read-write.ttl", this.username);
+        //perform POST  to child resource as non-admin
+        doPost(resourceUri, new Headers(new Header("Content-Type", "text/plain"),
+                                        new Header("Slug", "child2")),
```

Comment:
As @bseeger [notes](https://github.com/fcrepo/Fedora-API-Test-Suite/pull/150#discussion_r212671999), we may want to begin getting away from the use of `slugs`s.